src/ray/core_worker/fiber.h

```diff
@@ -101,7 +101,8 @@ class FiberState {
     return true;
   }
 
-  explicit FiberState(int max_concurrency)
+  explicit FiberState(int max_concurrency,
+                      std::function<std::function<void()>()> = nullptr)
```

Comment:
drop a comment about the unused arg

Reply:
update https://github.com/ray-project/ray/pull/52575/commits/1130e714951e80e3f2595c0f4095537d100c0aa5